yarn-project/circuit-types/src/logs/tx_l2_logs.ts

```diff
@@ -11,15 +14,15 @@ export class TxL2Logs {
     /**
      * An array containing logs emitted in individual function invocations in this tx.
      */
-    public readonly functionLogs: FunctionL2Logs[],
+    public readonly allFunctionLogs: FunctionL2Logs[],
```

Comment:
Why was this renamed?

Reply:
But you would have 
```
for (const functionLogs of this.functionLogs) {
```

Just seems weird to call it "all" when this is the only time we do it and the rest of the time we don't when dealing with collections 🤷 Seems inconsistent when we have `newNullifiers` and now `allNewNullifiers` etc